src/site/xdoc/server/config-batchsizes.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+<document>
+
+    <properties>
+        <title>Apache James Server 3 - Batch sizes Configuration</title>
+    </properties>
+
+    <body>
+
+        <section name="Batch sizes configuration">
+
+            <p>Limits on operations to be done at the same time. The goal is to avoid to overwhelm the database?</p>
+
+            <p>This file is for now specific to Cassandra guice wiring.</p>
+
+            <subsection name="batchsizes.properties">
+
+                <p>Consult <a href="https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra/destination/conf/batchsizes.properties">batchsizes.properties</a> in GIT to get some examples and hints.</p>
+
+                <br/>
+
+                <dl>
+                    <dt><strong>fetch.metadata</strong></dt>
+                    <dd>Maximum number of messages to read at the same time at the 'metadata' fetch level.</dd>
+
+                    <dt><strong>fetch.header</strong></dt>
+                    <dd>Maximum number of messages to read at the same time at the 'header' fetch level.</dd>
+
+                    <dt><strong>fetch.body</strong></dt>
+                    <dd>Maximum number of messages to read at the same time at the 'body' fetch level.</dd>
+
+                    <dt><strong>fetch.full</strong></dt>
+                    <dd>Maximum number of messages to read at the same time at the 'full' fetch level.</dd>
```

Comment:
maybe explaining that 'metadata', 'header', 'body' and 'full' are from an RFC could help to understand the meaning of these sentences